test/util/getBuildSignatureFunc.js

```diff
@@ -0,0 +1,16 @@
+const crypto = require('crypto')
+const config = require('~/src/config')
+
+config.load()
+const SHARED_SECRET = config.getGithubWebhookSecret()
+
+module.exports = function getBuildSignatureFunc (type) {
+  if (type !== 'GITHUB') {
```

Comment:
Related to comment below. "I think instead of passing the signatureBuilder, you should just require it in the files that need it. Bitbucket doesn't need one of these right now anyway."